src/core/public/chrome/ui/header/_index.scss

```diff
@@ -7,6 +7,10 @@
   z-index: 10;
 }
 
+.euiDataGrid__restrictBody .chrHeaderWrapper {
+  display: none;
+}
+
```

Comment:
FYI @snide CC: @andreadelrio , you've set `z-index` to 0, I migrated to `display: none` to solve the following overlapping issue when using the `Full screen` view. --

<img width="2311" alt="Bildschirmfoto 2020-06-15 um 15 37 47" src="https://user-images.githubusercontent.com/463851/84664372-bd6ae380-af1e-11ea-8be0-4ebe6c81b742.png">